imap_processing/idex/IDEXPacketParser.py

```diff
@@ -0,0 +1,120 @@
+import bitstring
+import numpy as np
+from space_packet_parser import parser, xtcedef
+from imap_processing import packet_definition_directory
+
+
+class IDEXPacketParser():
+    def __init__(self, idex_packet_file: str):
+
+        idex_xtce = f"{packet_definition_directory}/idex_packet_definition.xml"
+        idex_definition = xtcedef.XtcePacketDefinition(xtce_document=idex_xtce)
+        idex_parser = parser.PacketParser(idex_definition)
+
+        idex_binary_data = bitstring.ConstBitStream(filename=idex_packet_file)
+        idex_packet_generator = idex_parser.generator(idex_binary_data)
+
+        self.epochs = []
```

Comment:
I was wondering what is difference between `epochs` and `data`? This might relate to question I had about `scitype == 1`.

Reply:
"epochs" is a list of times that the dust events have occurred, and the data is the actual data from those events.  epoch is generally just the CDF/SPDFs term for "time".